autogen/logger/file_logger.py

```diff
@@ -25,6 +25,7 @@ class FileLogger(BaseLogger):
     def __init__(self, config: Dict[str, Any]):
         self.config = config
         self.session_id = str(uuid.uuid4())
+        self.thread_id = threading.get_ident()
```

Comment:
added `thread_id` 

reference : #2656

Reply:
The thread id should be obtained when logging each event

Reply:
updated this. Thanks